### DISCUSSION TITLE
i would appreciate a guide for how to migrate my xresources-based theme to rasi

### DISCUSSION THREAD

**Discussion by colons** [davatorium/rofi#1551] [Q&A]
i understand that ditching xrdb-based theme configuration was a reasonable thing to do, but i am really struggling to know what to do to make rofi look like it used to now that support for the old style has been removed

the docs for the theme stuff are kind of dense and complicated, and also contain caveats about potentially being out of date, and all i want to know is how to turn my old `.Xresources` entries into something that will work with rofi 1.7:

```
!                   bg        fg        bgalt     hlbg      hlfg
rofi.color-normal:  #222222,  #dddddd,  #222222,  #dd6666,  #222222
!                   bg        border    separator
rofi.color-window:  #222222,  #dd6666,  #222222
```

in my imagination, such a guide would be pretty short, but im not sure. it certainly seems to me like it'd be a helpful addition to the docs

**Comment by DaveDavenport** [comment]
Do you mean? https://github.com/davatorium/rofi/discussions/1400

**Reply by DaveDavenport** [reply]
The manpages should be up2date.  I do not have time to work on the wiki and it seems there is no help for this job.
Regrettable it is what it is, but I cannot afford to spend more time on rofi then I already do.

**Reply by colons** [reply]
oh heck, maybe; i guess i can try to install an old version and try this

**Reply by colons** [reply]
no, that's entirely fair! rofi is great and you're putting enough work in as it is, i just wasn't sure what was expected of me here as someone who just wanted continuity

**Reply by DaveDavenport** [reply]
given the themes has not much changed you can try dumping the default theme:
```bash
rofi -no-config -dump-theme >> ~/.config/rofi/mytheme.rasi
```
edit the entries on the top and use that by adding:
```css
@theme "mytheme"
```
at the end of your ~/.config/rofi/config.rasi file.

**Reply by DaveDavenport** [reply]
It is still structured to for the old 'conversion hack'.
(also here: https://github.com/davatorium/rofi/blob/next/doc/default_theme.rasi)

**Reply by colons** [reply]
thank you for the help; i got something working that's pretty similar to what i had
